src/components/TrackView.js

```diff
@@ -1,13 +1,14 @@
 import React, {Component} from "react";
-import {Search} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
-import {Timeline, TimelineEvent} from 'react-event-timeline'
+import {Timeline, TimelineEvent} from 'react-event-timeline';
 import {connect} from "react-redux";
 import RiderSearch from "./RiderSearch";
+import SearchResult from "./SearchResult";
 
 class TrackView extends Component {
     render() {
       const {judgments} = this.props;
+        const {search} = this.props;
       const {raceGroups} = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "raceGroups" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'raceGroups' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)